modules/ContextUtils.js

```diff
@@ -111,8 +112,12 @@ export function ContextSubscriber(name) {
       this[unsubscribeKey] = null
     },
 
-    [handleContextUpdateKey](eventIndex) {
+    [handleContextUpdateKey](eventIndex, nextValue) {
       if (eventIndex !== this.state[lastRenderedEventIndexKey]) {
+        if (this.context[name] !== nextValue) {
+          // React uses a stale value so we update it manually.
+          this.context[name] = nextValue;
```

Comment:
I’m not proud of this but it works fine. I’ll keep an eye on this not breaking in the React repo 😄